inst/Upgrades.qml

```diff
@@ -0,0 +1,287 @@
+		ChangeJS
+		{
+			name:		"alternative"
+			jsFunction:	function(options)
+			{
+				switch(options["alternative"])
+				{
+					case "notEqualToTestValue":		return "twoSided";
+					case "greaterThanTestValue":	return "greater";
+					case "lessThanTestValue":		return "less";
+				}
+			}
+		}
+		ChangeRename{	from: "confidenceInterval";						to: "ci"						}
+		ChangeRename{	from: "confidenceIntervalInterval";				to: "ciLevel"					}
+		ChangeRename{	from: "VovkSellkeMPR";							to: "vovkSellke"				}
+		ChangeRename{	from: "descriptivesPlots";						to: "descriptivesPlot"			}
+		ChangeRename{	from: "descriptivesPlotsConfidenceInterval";	to: "descriptivesPlotCiLevel"	}
+	}
+
+	Upgrade
+	{
+		functionName: 		"BinomialTestBayesian"
+		fromVersion:		"0.16.4"
+		toVersion:			"0.17.0"
+
+		ChangeRename{	from: "hypothesis";	to: "alternative"	}
+		ChangeJS
+		{
+			name:		"alternative"
+			jsFunction:	function(options)
+			{
+				switch(options["alternative"])
+				{
+					case "notEqualToTestValue":		return "twoSided";
+					case "greaterThanTestValue":	return "greater";
+					case "lessThanTestValue":		return "less";
+				}
+			}
+		}
+		ChangeRename{	from: "plotPriorAndPosterior";					to: "priorPosteriorPlot"					}
+		ChangeRename{	from: "plotPriorAndPosteriorAdditionalInfo";	to: "priorPosteriorPlotAdditionalInfo"	}
+		ChangeRename{	from: "plotSequentialAnalysis";					to: "bfSequentialPlot"				}
+		ChangeRename{	from: "descriptivesPlots";						to: "descriptivesPlot"						}
+		ChangeRename{	from: "descriptivesPlotsCredibleInterval";		to: "descriptivesPlotCiLevel"				}
+	}
+
+	Upgrade
+	{
+		functionName:		"MultinomialTest"
+		fromVersion:		"0.16.4"
+		toVersion:			"0.17.0"
+
+		// this option did not do anything and was always hidden so now it's removed
+		ChangeRemove{	name: "simulatepval"	}
+		ChangeRename{	from: "counts";					to: "count"				}
+		ChangeRename{	from: "exProbVar";				to: "expectedCount"		}
+		ChangeRename{	from: "hypothesis";				to: "testValues"		}
+		ChangeJS
+		{
+			name:		"testValues"
+			jsFunction:	function(options)
+			{
+				switch(options["testValues"])
+				{
+					case "multinomialTest":		return "equal";
+					case "expectedProbs":		return "custom";
+				}
+			}
+		}
+		ChangeRename{	from: "tableWidget";					to: "testValuesCustom"			}
+		ChangeRename{	from: "VovkSellkeMPR";					to: "vovkSellke"				}
+		ChangeRename{	from: "descriptives";					to: "descriptivesTable"			}
+		ChangeRename{	from: "confidenceInterval";				to: "descriptivesTableCi"		}
+		ChangeRename{	from: "confidenceIntervalInterval";		to: "descriptivesTableCiLevel"	}
+		ChangeRename{	from: "countProp";						to: "descriptivesType"			}
+		ChangeJS
+		{
+			name: "descriptivesType"
+			jsFunction:	function(options)
+			{
+				switch(options["descriptivesType"])
+				{
+					case "descCounts":		return "counts";
+					case "descProps":		return "proportions";
+				}
+			}
+		}
+		ChangeRename{	from: "descriptivesPlot";						to: "descriptivesPlot"			}
```

Comment:
There's no change here is there? Was there an s after ..Plot??

Reply:
Ah yeah, this isn't changed

Reply:
ah yeah, for the multinomial it was already `descriptivesPlot`, only for binomial it was `descriptivesPlots`. So for the multinomial we don't need the upgrade